pull.md

```diff
@@ -0,0 +1,101 @@
+# Pull
+
+## Goal
+
+Give users a way to pull resources from one cluster into another. Pulling
+should take advantage of preexisting state in puller so that data doesn't need
+to be transferred twice. Pulling should be robust in the face of network issues
+sustained network failures can prevent a pull from succeeding, but network
+hiccups shouldn't. Pulling will often require transferring very large amounts
+of data, so a network hiccup shouldn't be able to break the entire process, it
+should be smart about retrying.
```

Comment:
To me it's strongly implied that data will flow from the source cluster directly to the destination cluster; not through the workstation that pachctl is running on.  But I don't see that explicitly written out anywhere, so some clarification might be good?

Reply:
Will add clarification, your understanding is correct though.